easybuild/easyconfigs/l/Libint/Libint-2.0.3-foss-2018b.eb

```diff
@@ -0,0 +1,22 @@
+name = 'Libint'
+version = '2.0.3'
+
+homepage = 'https://sourceforge.net/p/libint'
+description = """Libint library is used to evaluate the traditional (electron repulsion) and certain novel two-body
+ matrix elements (integrals) over Cartesian Gaussian functions used in modern atomic and molecular theory."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://downloads.sourceforge.net/project/libint/libint-for-mpqc']
+sources = ['libint-%(version)s-stable.tgz']
+# Remove some problematic (and not very useful) FMA header code that does not compile with GCC.
+# This is done with a patch, because it also affects anything that links against Libint.
+# This way, FMA compilert optimization can remain enabled.
```

Comment:
@tovrstra Please move this descriptive comment to the top of the patch file, and mention yourself as author.

Any lines before the first line (`--- ...`) in the patch file are comments, no need to escape with `# ` .

Keep in mind that you'll need to update the checksum for the patch accordingly.

Reply:
Done